frontend/src/scenes/trends/viz/ActionsBarValueGraph.tsx

```diff
@@ -37,13 +37,13 @@ export function ActionsBarValueGraph({
         _data.sort((a, b) => b.aggregated_value - a.aggregated_value)
         const colorList = getChartColors(color)
         const days = results.length > 0 ? results[0].days : []
-
         setData([
             {
                 labels: _data.map((item) => item.label),
                 data: _data.map((item) => item.aggregated_value),
                 actions: _data.map((item) => item.action),
                 days,
+                breakdown_values: _data.map((item) => item.breakdown_value),
```

Comment:
Kind of a nit, but what about sticking with the camelCase syntax to be in line with the other keys?